src/development/tools/devtools/performance.md

```diff
@@ -85,6 +85,16 @@ experience UI jank or dropped frames.
 For more information on how to analyze your app's performance,
 see [Flutter performance profiling][].
 
+### Shader compilation
+Shader compilation occurs when a shader is first used in your Flutter
+app. Frames that are performing shader compilation are marked in dark
+red:
+
+![Screenshot of shader compilation for a frame]({{site.url}}/assets/images/docs/tools/devtools/shader-compilation-frames-chart.png)
```

Comment:
I would use a different screenshot that shows a janky frame with shader compilation time. A warning icon will show up if the frame is actually janky. Also could be good to show the tooltip when you hover over the janky frame, which will show how much time on the raster thread was spent compiling shaders.

Reply:
@kenzieschmoll SGTM

@sfshaza2 When I stage the site it tries to load the image from docs.flutter.dev, and it doesn't exist yet. I think I can edit _config.yml to change site.url to point to my staging site, though.

Reply:
Ah, I hadn't run into this yet. Don't worry about it!